src/writer/settings.ts

```diff
@@ -21,6 +33,38 @@ export async function verify(body): Promise<any> {
   });
   const isController = controller === body.address;
   const space = await getSpace(msg.space, true);
+
+  if (network !== 'testnet') {
+    const hasTicket = msg.payload.strategies.some(strategy => strategy.name === 'ticket');
+    const hasVotingValidation =
+      msg.payload.voteValidation?.name && !['any'].includes(msg.payload.voteValidation.name);
+
+    if (hasTicket && !hasVotingValidation) {
+      return Promise.reject('space with ticket requires voting validation');
+    }
+
+    const hasProposalValidation =
+      (msg.payload.validation?.name && msg.payload.validation.name !== 'any') ||
+      msg.payload.filters?.minScore ||
+      msg.payload.filters?.onlyMembers;
+
+    if (!hasProposalValidation) {
+      return Promise.reject('space missing proposal validation');
+    }
+  }
+
+  if (
+    !Object.values(networks)
+      .map(n => n.key)
+      .includes(msg.payload.network)
+  ) {
+    return Promise.reject('invalid network');
+  }
+
+  if (!getSupportedNetworks(network).includes(msg.payload.network)) {
+    return Promise.reject('wrong network');
+  }
```

Comment:
Not going to handle with schema? https://github.com/snapshot-labs/snapshot.js/pull/936